packages/features/insights/server/events.ts

```diff
@@ -22,13 +22,18 @@ class EventsInsights {
         if (typeof item.timeStatus === "string" && item) {
           aggregate[item.timeStatus] += item?._count?._all ?? 0;
           aggregate["_all"] += item?._count?._all ?? 0;
+
+          if (item.noShowHost) {
```

Comment:
We add to the separate noShowHost "bucket" to keep track of the total count.

Reply:
<img width="367" alt="Screenshot 2024-09-11 at 10 50 08 AM" src="https://github.com/user-attachments/assets/521dbb8d-2f72-4779-a85f-2cafe6c62e0b">